Add tasks on Bell states changes to Multi-Qubit States kata (#1385)
microsoft/qsharp: katas/content/multi_qubit_systems/bell_state_change_3/solution.md

> We remember from Task "Sign flip" that the Pauli-Z gate leaves the sign of the $|0\rangle$ component of the single qubit superposition unchanged but flips the sign of the $|1\rangle$ component of the superposition. We have also just seen in"Bell state change 2" how to change our input state to the state $\frac{1}{\sqrt{2}} \big(|01\rangle + |10\rangle\big)$, which is almost our goal state (disregarding the phase change for the moment). So it would seem that a combination of these two gates will be what we need here. The remaining question is in what order to apply them, and to which qubit.

Reviewer: ```suggestion
We remember from the Single-Qubit Gates kata that the Pauli Z gate leaves the sign of the $|0\rangle$ component of the single qubit superposition unchanged but flips the sign of the $|1\rangle$ component of the superposition. We have also just seen in "Bell State Change 2" how to change our input state to the state $\frac{1}{\sqrt{2}} \big(|01\rangle + |10\rangle\big)$, which is almost our goal state (disregarding the phase change for the moment). So it would seem that a combination of these two gates will be what we need here. The remaining question is in what order to apply them, and to which qubit.
```

> First of all, which qubit? Looking back at"Bell state change 2", it seems clear that we need to use qubit `qs[0]`, like we did there.

Reviewer: ```suggestion
First of all, which qubit? Looking back at the task "Bell state change 2", it seems clear that we need to use qubit `qs[0]`, like we did there.
```

> Second, in what order should we apply the gates? Remember that the Pauli-Z gate flips the phase of the $|1\rangle$ component of the superposition and leaves the $|0\rangle$ component alone.  

Reviewer: ```suggestion
Second, in what order should we apply the gates? Remember that the Pauli Z gate flips the phase of the $|1\rangle$ component of the superposition and leaves the $|0\rangle$ component alone.
```
